cmd/substreams/pack.go

```diff
@@ -20,12 +23,45 @@ var packCmd = &cobra.Command{
 
 func init() {
 	rootCmd.AddCommand(packCmd)
+	packCmd.Flags().StringP("output-dir", "o", ".", cli.FlagDescription(`
+		Optional flag to specify output directory to output generated .spkg file. 
+		If the received argument starts with a "." it will be treated as a local path relative to the supplied manifest path.
+	`))
+}
+
+func is_relative_path(path string) bool {
```

Comment:
Golang uses `camelCase` for private method and `PascalCase` for public methods, so should have been `isRelativePath`.

Second, this method is not needed, call site should be replace with `!filepath.IsAbs(path)`

Reply:
Have changed, thanks for the suggestion